### PR TITLE
K8SPXC-476 - Ignore prerelease in semver

### PR DESCRIPTION
**Pull request by blez**

### CODE REVIEW

**Comment by JNKPercona** [comment]
| Test name  | Status |
| ------------- | ------------- |
| api-tests | passed |

**Comment by JNKPercona** [comment]
| Test name  | Status |
| ------------- | ------------- |
| api-tests | passed |

**Comment by JNKPercona** [comment]
| Test name  | Status |
| ------------- | ------------- |
| api-tests | passed |

**Review by tplavcic — APPROVED** [review]

File: server/filters.go [hunk 1]
```diff
@@ -107,11 +107,12 @@ func pxcFilter(versions map[string]*pbVersion.Version, apply string, current str
 		}
 
 		for _, s := range sorted {
-			if s.Equal(c) || s.LessThan(c) {
+			if s.Equal(c) || (s.Major() < c.Major() && s.Minor() < c.Minor() && s.Patch() < c.Patch()) {
```
Comment (delgod) [inline server/filters.go:110]: maybe will it be better?
```suggestion
			if s.Major() = c.Major() && s.Minor() = c.Minor() && s.Patch() = c.Patch() {
```

**Review by delgod — CHANGES_REQUESTED** [review]

**Comment by JNKPercona** [comment]
| Test name  | Status |
| ------------- | ------------- |
| api-tests | passed |

**Review by delgod — APPROVED** [review]

**Pull request merged**